aptos-move/framework/aptos-framework/sources/configs/staking_config.spec.move

```diff
@@ -83,6 +83,7 @@ spec aptos_framework::staking_config {
     }
 
     spec calculate_and_save_latest_epoch_rewards_rate(): FixedPoint64 {
+        pragma verify_duration_estimate = 120;
```

Comment:
what is this for?

Reply:
Yesterday, I got some move prover test errors in Github actions. I think Junkil added this to fix those errors.